ui/components/K8sContextSwitcher/K8sContextSwitcher.styles.js

```diff
@@ -0,0 +1,120 @@
+import makeStyles from "@mui/styles/makeStyles";
```

Comment:
We are refactoring , upgrade from mui4 to mui5 , makeStyles is from mui4

Reply:
I think makestyles is there in mui5 too. But imported using `@mui/styles/makeStyles` instead of `@mui/material/styles`([Reference](https://mui.com/material-ui/migration/v5-style-changes/#%E2%9C%85-update-makestyles-import)). Am I misinterpreting your comment?

Reply:
@SAEb-ai  Yup i know that but we need to move away from using makeStyles, so you should look at [styled APi](https://mui.com/system/styled/) or use plain css, we are using styled in many other components so please refer them